### PR TITLE
Project Submision

### PR DESCRIPTION
**Pull request by victoriensukarieh**

### CODE REVIEW

File: victoriensukarieh.Calculator/ConsoleCalculator/Models/Operation.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace ConsoleCalculator.Models;
```
Comment (TheCSharpAcademy) [inline victoriensukarieh.Calculator/ConsoleCalculator/Models/Operation.cs:1]: Please tidy up code formating: This button in the bottom of the code window is very useful for that: 😁
image

File: victoriensukarieh.Calculator/ConsoleCalculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        bool useSaved = false;
+
+        // Display title as the C# console calculator app.
+        Console.WriteLine("Console Calculator in C#\r");
+        Console.WriteLine("------------------------\n");
+        Calculator calculator = new Calculator();
+
+        while (!endApp)
+        {
+            // Declare variables and set to empty or zeros.
```
Comment (TheCSharpAcademy) [inline victoriensukarieh.Calculator/ConsoleCalculator/Program.cs:20]: Please remove all comments that can be explained by the code itself.

File: victoriensukarieh.Calculator/ConsoleCalculator/Program.cs [hunk 3]
```diff
@@ -0,0 +1,181 @@
+﻿using CalculatorLibrary;
+using ConsoleCalculator.Models;
```
Comment (TheCSharpAcademy) [inline victoriensukarieh.Calculator/ConsoleCalculator/Program.cs:2]: Please remove unecessary usings and add spacing between usings and namespace, and between namespace and class name, as per code conventions: 

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@victoriensukarieh The project looks great and everything works fine. I like how you split the menu per number of operands. 😊

Since this is your second project, I'll be more strict in enforcing the removal of unnecessary comments and general code tidiness.

**Comment by victoriensukarieh** [comment]
@TheCSharpAcademy i am trying my best. i want to understand oop more and i wanna reach a point where starting an application and finishing it becomes automatic

**Comment by victoriensukarieh** [comment]
i resolved the final issues and all is done @TheCSharpAcademy

**Pull request merged**